src/zcl_alog_progress_rep_logger.clas.abap

```diff
@@ -53,4 +76,14 @@ CLASS zcl_alog_progress_rep_logger IMPLEMENTATION.
                                               i_processed          = 1
                                               i_total              = 100 ).
```

Comment:
These should also use `mv_actual_entries` and `mv_total_entries`, shouldn't they?

Reply:
yes, you are right, there are two point and I change only one.